srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml

```diff
@@ -875,6 +875,12 @@ SRTP packets                 <------->      SRTP packets
         allocation of even values over odd ones until the even code
         points are consumed to avoid conflicts with pre standard
         versions of EKT that have been deployed.</t>
+
+        <t> All new EKT message MUST be defined to have a length as
+        second from the last element so that implementations that do
+        not understand the new extension can still skip over it when
+        parsing the message. <t>
```

Comment:
Actually, having the length there for extensions is only one reason. Another is that this obviates the need for successful processing to be dependent on having knowledge of the SRTP master key length.  While this is usually known by receiving endpoint, some scenarios we discussed in PERC, like having a null HBH cipher presented problems for media distributors.